packages/modular-scripts/DO_NOT_IMPORT_THIS_OR_YOU_WILL_BE_FIRED_craco.config.js

```diff
@@ -67,95 +71,81 @@ module.exports = {
         } catch (err) {
-          console.info(
-            "It appears you're using `@finos/perspective`. Run `yarn install @finos/perspective-webpack-plugin`, and modular will automatically optimise its bundling to be more efficient as described in https://github.com/finos/perspective/tree/master/packages/perspective-webpack-plugin.",
-          );
-          throw err;
+          /* silently fail */
         }
-        // both perspective and its webpack plugin are available, let's
-        // add it to our webpack pipeline.
-        const PerspectivePlugin = require('@finos/perspective-webpack-plugin');
-        webpackConfig.plugins.push(new PerspectivePlugin());
-      } catch (err) {
-        /* silently fail */
-      }
+      });
 
       return webpackConfig;
     },
   },
   jest: {
     configure(jestConfig) {
-      const options = {
-        loaders: {
-          '.js': 'jsx',
-          '.test.js': 'jsx',
-          '.ts': 'tsx',
-          '.test.ts': 'tsx',
-        },
-      };
-
-      const transform = { ...jestConfig.transform };
-
-      // Replace babel transform with esbuild
-      // babelTransform is first transformer key
-      /* 
-      transform:
-        {
-          '^.+\\.(js|jsx|mjs|cjs|ts|tsx)$': 'node_modules\\react-scripts\\config\\jest\\babelTransform.js',
-          '^.+\\.css$': 'node_modules\\react-scripts\\config\\jest\\cssTransform.js',
-          '^(?!.*\\.(js|jsx|mjs|cjs|ts|tsx|css|json)$)': 'node_modules\\react-scripts\\config\\jest\\fileTransform.js'
-        }
-      */
-      const babelKey = Object.keys(transform)[0];
-
-      // We replace babelTransform and add loaders to esbuild-jest
-      transform[babelKey] = [require.resolve('esbuild-jest'), options];
-
-      // Adds loader to all other transform options (2 in this case: cssTransform and fileTransform)
-      // Reason for this is esbuild-jest plugin. It considers only loaders or other options from the last transformer
-      // You can see it for yourself in: /node_modules/esbuild-jest/esbuid-jest.js:21 getOptions method
-      // also in process method line 32 gives empty loaders, because options is already empty object
-      // Issue reported here: https://github.com/aelbore/esbuild-jest/issues/18
-      Object.keys(transform).forEach((key) => {
-        if (babelKey === key) return; // ebuild-jest transform, already has loader
-
-        // Checks if value is array, usually it's not
-        // Our example is above on 70-72 lines. Usually default is: {"\\.[jt]sx?$": "babel-jest"}
-        // (https://jestjs.io/docs/en/configuration#transform-objectstring-pathtotransformer--pathtotransformer-object)
-        // But we have to cover all the cases
-        if (Array.isArray(transform[key]) && transform[key].length === 1) {
-          transform[key].push(options);
-        } else {
-          transform[key] = [transform[key], options];
-        }
-      });
-
       return {
         ...jestConfig,
```

Comment:
Potentially at this point do we just completely ignore what's coming from craco before we lift this to running jest ourselves?